components/tabBarComponents/EventTabBar.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react"
+import { StyleSheet, TouchableOpacity, View } from "react-native"
+import { Ionicons } from "@expo/vector-icons"
+import { BlurView } from "expo-blur"
+
+export const EventTabBar = () => {
```

Comment:
Rename to "NavigationTabBar" or something similar, since this will be used for navigation across the whole app